upgrade/network/network_upgrade_1.2_to_1.3.md

```diff
@@ -0,0 +1,125 @@
+
+- Removal of CMN VLAN from UAN to complete BICAN cut-over
+- Addition of ACLS to prevent CAN traffic on the CMN
+- Addition of ACLs to prevent cross-network load balancer access: NMN to HMNLB and HMN to NMNLB
+
+Switch configuration upgrades for CSM 1.3 should be completed by a networking subject matter expert.
+While addition of ACLs is a relatively safe process, the migration of user traffic from the management network CAN to the high speed network CHN is sequence dependent and not without risk.
+Strict attention to the cut-over sequence will minimize downtime to UAN NCNs.
+
+## When to perform switch upgrades
+
+At the end of the CSM 1.3 upgrade process.
+
+Unlike most upgrades, the CSM 1.2 management network switch configuration changes come at the end of the system upgrade, not at the beginning. This allows the continued operations of services during upgrade.
+Additionally, UAN can be operated normally and any reboot or rebuild operations can be scheduled between between administrators and UAN users.
+
+## Prerequisites
+
+- **IMPORTANT** If conversion from `CAN` to `CHN` is required, the [SLS update procedure](chn_enable.md) must have been run at the beginning of the upgrade process.
+- An up-to-date and validated copy of the system [CCJ/Paddle JSON topology file](https://github.com/Cray-HPE/canu/tree/main#validate-paddle) (preferred), or an up-to-date and validated [SHCD spreadsheet](https://github.com/Cray-HPE/canu/tree/main#validate-shcd).
+  - The SHCD should be used as starting data over the CCJ/Paddle file only when node, network or cabling changes have taken place on the system.
+  - A correct cabling and network topology must be reflected to ensure working switch configurations.
+- An [up-to-date custom configurations](https://github.com/Cray-HPE/canu/tree/main#generate-switch-configs-including-custom-configurations) `YAML` file to be used in `CANU` switch configuration generation with the `--custom-config` flag.
+  - These custom configurations are critical to preserve site uplinks and any configurations which extend beyond plan-of-record.
+  - This file should have been created during the CSM 1.2 system install or upgrade process and is required to ensure working switch configurations.
+
+## Upgrade CANU to the latest version
+
+- In a browser, navigate to [CANU Releases](https://github.com/Cray-HPE/canu/releases) and copy the latest release RPM URL
+- (`ncn-m001#`) Download the latest CANU RPM with the release RPM URL
+- (`ncn-m001#`) Upgrade CANU with `rpm -Uvh <release RPM URL>`
+
+## Backup running switch configurations
+
+- (`ncn-m001#`) Use CANU to backup the switch running configurations.
+
+    Enter the switch administrative password when prompted.
+
+     ```bash
+     mkdir switch-configs-csm-1.3
+     cd switch-configs-csm-1.3
+     canu backup network --folder 1.2
```

Comment:
What if they are doing this CAN->CHN migration after a 1.3 upgrade?   Will the 1.2 folder be confusing?

Reply:
This is the generalized 1.2 to 1.3 upgrade procedure.  The use cases I see are:
1. 1.3 fresh install - will never get to this doc, nor the can->chn doc and switches will be configured early as normal.
2. 1.3 upgrade can->can will never hit the can->chn doc and the upgrade will end the switch updates for this release.
3. 1.3 upgrade chn->chn will never hit the can->chn doc and the upgrade will end the switch updates for this release.
4. 1.3 upgrade can->chn during upgrade will hit the chn_enable doc and then be passed here as the last step.
5. 1.3 upgrade complete.. then apply chn_enable procedure. they would upgrade can->can and then run the procedure so see item 2.  

will put some verbiage in to cover use case 5.